ts/components/Avatar.tsx

```diff
@@ -140,4 +138,16 @@ export class Avatar extends React.Component<Props, State> {
       </div>
     );
   }
+
+  private getBorderStyle(color?: string, width?: number) {
+    const borderWidth = typeof width === 'number' ? width : 3;
```

Comment:
While the operator precedence is correct, the readability can be improved a tiny notch:
```suggestion
    const borderWidth = (typeof width === 'number') ? width : 3;
```

Reply:
actually tslint removed those brackets for me :/

Reply:
the machines are winning!